workspace-manager-clienttests/src/main/resources/configs/integration/BasicAuthenticated.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "BasicAuthenticated",
+  "description": "Get workspace once. Authentication required.",
+  "serverSpecificationFile": "workspace-dev.json",
+  "kubernetes": {},
+  "application": {},
+  "testScripts": [
+    {
+      "name": "GetWorkspace",
+      "numberOfUserJourneyThreadsToRun": 1,
```

Comment:
I thought this was 10 before.

Reply:
Yes, this was changed from 10 to 1 when I separate out the configs into `integration` and `perf`.